lib/sandbox-api-hack.js

```diff
@@ -7,7 +7,10 @@ var path = require('path');
 var is = require('is_js');
 
 module.exports = function (gulp, gutil) {
-  var SUPPORTED_UPDATE_PLUGIN_POINTS = ['skins', 'quilts', 'layouts', 'lang', 'components'];
+  var SUPPORTED_UPDATE_PLUGIN_POINTS = [{
+    id: 'skins',
+    sub: ['images']
```

Comment:
should this be 
sub: [{id: 'images'}]
in case we ever need to support deeper nesting